### PR TITLE
Service worker guide

### PR DESCRIPTION
**Pull request by CrystalOnScript**
Updated amp-as-pwa.html to include information about AMP Service Worker. Removed manifest information as PWA will be covered in tutorial, once it has been published. This merge can wait until the tutorial is up.

### CODE REVIEW

**Comment by CrystalOnScript** [comment]
+Tutorial

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+…
+<script async custom-element="amp-install-serviceworker" src="https://cdn.ampproject.org/v0/amp-install-serviceworker-0.1.js"></script>
+…
+</head>
+<body>
+   <amp-install-serviceworker src="/sw.js"
+           data-iframe-src="install-sw.html"
+           layout="nodisplay">
+   </amp-install-serviceworker>
+...
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md]: nit: the `...` imply that the element needs to be at the top of the body, which is not the case. I'd suggest instead moving it to the bottom. It's not critical at all, probably only a matter of taste, but I usually suggest moving things like `amp-install-serviceworker`, `amp-analytics`, ... to the bottom of the page.

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+## Prefetch Links
+
+The AMP Service Worker prefetches links that have the `data-rel=prefetch` attribute. 
+This enables users to view pages offline even if they haven't visited them yet. We'll add the attribute to our link tag for `lineup.html`.
+
+
+[sourcecode:html]
+...
+<a href="/lineup.html" data-rel="prefetch">See Full Lineup</a>
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md]: sidenote: we should use this approach for step-by-step guides in amp.dev

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+…
+</head>
+<body>
+   <amp-install-serviceworker src="/sw.js"
+           data-iframe-src="install-sw.html"
+           layout="nodisplay">
+   </amp-install-serviceworker>
+...
+[/sourcecode]
+
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md]: Suggestion: add a callout explaining that the service worker should be served from the root

```
[tip type="important"]
 **Important –** the service worker needs to be served from the root directory (`/serviceworker.js`) to be able to cache all the content of your site.
 [/tip]
```

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+# Add the AMP Service Worker
+
+A service worker is a script that your browser runs in the background, 
+separate from a web page, that extends the browsers features by caching requests to improve performance and providing offline functionality. 
+Building a service worker from scratch is possible but time consuming. 
+Libraries like Workbox help, but AMP goes one step further by offering the [AMP Service Worke](https://github.com/ampproject/amp-sw)r, 
+in which AMP automates a lot of steps directly, including the caching of AMP Scripts, assets and documents.
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md]: Suggestion add somewhere: ".. and implements common best practices such as [navigation preload](https://developers.google.com/web/updates/2017/02/navigation-preload)"

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/advertise_amp_stories.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 $title: Advertise in AMP Stories
-$order: 3
+$order: 13
```
[the remark this answers is not in the document]
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/advertise_amp_stories.md]: Ahh,nope! Whoops - thanks for the catch

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md (hunk 2 above)
Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md]: What part of the approach? All on one page?

**Comment by CrystalOnScript** [comment]
Updated per suggestions, @sebastianbenz PTAL :)

File: pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md [hunk 6]
```diff
@@ -71,8 +71,9 @@ Add a file titled `manifest.json` to your repository with the following code:
 A service worker is a script that your browser runs in the background, 
 separate from a web page, that extends the browsers features by caching requests to improve performance and providing offline functionality. 
 Building a service worker from scratch is possible but time consuming. 
-Libraries like Workbox help, but AMP goes one step further by offering the [AMP Service Worke](https://github.com/ampproject/amp-sw)r, 
-in which AMP automates a lot of steps directly, including the caching of AMP Scripts, assets and documents.
+Libraries like Workbox help, but AMP goes one step further by offering the [AMP Service Worker](https://github.com/ampproject/amp-sw), 
+in which AMP automates a lot of steps directly, including the caching of AMP Scripts, assets and documents
+and implements common best practices such as [navigation preload](https://developers.google.com/web/updates/2017/02/navigation-preload).
```
Comment (sebastianbenz) [inline pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md]: suggestion: s/and implement/as well as implementing/

Comment (CrystalOnScript) [reply pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/amp_to_pwa.md]: Done! thanks!

**Pull request merged**